### PR TITLE
add a component UserRepoDate

### PR DESCRIPTION
**Pull request by christopheluc62**

### CODE REVIEW

**Review by Hacene010 — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Solve conflicts before assigning me please

File: src/components/UserStarsRepo.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@
     border-style: solid;
     width: 180px;
     height: auto;
-    background-color: rgba($color: #232529, $alpha: 0.8);
+    background-color: rgba($color: #617aac, $alpha: 0.8);
```
Comment (LoicBrassart) [inline src/components/UserStarsRepo.scss]: I'm not sure if this works, and if it does I'd like to know how

File: src/components/UserRepoDate.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    });
+  }, []);
+
+  return (
+    <div>
+      <p>Liste des 5 repos les plus récents(nom-date-de création-langage)</p>
+      <section className="UserStarsRepo">
+        <div className="cardeList">
+          {repolist
+            .sort(function compare(a, b) {
```
Comment (LoicBrassart) [inline src/components/UserRepoDate.jsx:20]: You can drop the "compare" function name, it can be a nameless function

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some questionable code, but nothing too hard to change

File: src/components/UserStarsRepo.scss (hunk 1 above)
Comment (christopheluc62) [reply src/components/UserStarsRepo.scss]: background-color modified

**Review by christopheluc62 — COMMENTED** [review]
background-color modified

File: src/components/UserRepoDate.jsx (hunk 2 above)
Comment (christopheluc62) [reply src/components/UserRepoDate.jsx:20]: "compare" deleted

**Review by Hacene010 — DISMISSED** [review]

**Review by Hacene010 — DISMISSED** [review]

**Review by Hacene010 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**